src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -112,6 +144,102 @@ public static MarkdownPipelineBuilder Use(this MarkdownPipelineBuilder builder,
             return builder;
         }
 
+        public static string? GetElementType(MarkdownObject node)
+        {
+            if (node is null)
```

Comment:
with nullable checks turned on, there is no need to check `null` here as the parameter type is marked as `non-nullable`